sys/include/net/nanocoap.h

```diff
@@ -26,6 +26,25 @@
  * capital precede lower case). nanocoap provides the
  * COAP_WELL_KNOWN_CORE_DEFAULT_HANDLER entry for `/.well-known/core`.
  *
+ * ### Path matching ###
+ * By default the URI-path of an incoming request should match exactly the one
```

Comment:
Wording is a little off. I would expect to see the version below.

By default the URI-path of an incoming request should match exactly one of the registered resources.